arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -0,0 +1,1548 @@
+        },
+        "type": {
+          "description": "Resource type represents the complete path of the form Namespace/ResourceType/ResourceType/...",
+          "type": "string"
+        },
+        "location": {
+          "description": "Resource location",
+          "type": "string"
+        },
+        "tags": {
+          "description": "Resource tags",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "eTag": {
+          "description": "Optional ETag",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/ProtectedItem",
+          "description": "Resource properties"
+        }
+      }
+    },
+    "ProtectedItem": {
+      "description": "Base class for backup items.",
+      "type": "object",
+      "properties": {
+        "protectedItemType": {
+          "description": "backup item type",
+          "type": "string"
+        },
+        "backupManagementType": {
+          "description": "Type of backup managemenent for the backed up item. Possible values: AzureIaasVM, DPM or MAB",
+          "type": "string"
+        },
+        "workloadType": {
+          "description": "Type of workload this item represents. Possible values: VM, FileFolder.",
+          "type": "string"
+        },
+        "sourceResourceId": {
+          "description": "ARM ID of the resource to be backed up.",
+          "type": "string"
+        },
+        "policyId": {
+          "description": "ID of the backup policy with which this item is backed up.",
+          "type": "string"
+        },
+        "lastRecoveryPoint": {
+          "format": "date-time",
+          "description": "Timestamp when the last (latest) backup copy was created for this backup item.",
+          "type": "string"
+        }
+      },
+      "discriminator": "protectedItemType"
+    },
+    "ProtectedItemQueryObject": {
+      "description": "Filters to list backup items.",
+      "type": "object",
+      "properties": {
+        "backupManagementType": {
+          "description": "Backup management type for the backed up item. Possible values: AzureIaasVM, MAB, DPM.",
+          "type": "string"
+        },
+        "itemType": {
+          "description": "Type of workload this item represents. \r\n            Possible values: VM, FileFolder.",
+          "type": "string"
+        },
+        "policyName": {
+          "description": "Backup policy name associated with the backup item.",
+          "type": "string"
+        },
+        "containerName": {
+          "description": "Name of the container.",
+          "type": "string"
+        }
+      }
+    },
+    "GetProtectedItemQueryObject": {
+      "description": "Filters to list backup items.",
+      "type": "object",
+      "properties": {
+        "expand": {
+          "description": "Specifies if the additional information should be provided for this item.",
+          "type": "string"
```

Comment:
@amarzavery The expand query parameter here just takes True or False. When expand is True, the object returned would contain some extended properties.